std_msgs/Header header
crazyswarm_application/PreObservation[] pre_obs